src/Glue.elm

```diff
@@ -123,10 +126,11 @@ poly :
     , update : subMsg -> subModel -> ( subModel, Cmd msg )
     , subscriptions : subModel -> Sub msg
     }
-    -> Glue model subModel msg subMsg
+    -> Glue model subModel msg subMsg msg
 poly { get, set, init, update, subscriptions } =
     Glue
-        { get = get
+        { msg = identity
```

Comment:
Wow, shame on me, I'm still learning, but on my defense I never need it in OOP before :D